### PR TITLE
fix user_info

### PR DESCRIPTION
**Pull request by drowsy810301**
fix user_info.py

### CODE REVIEW

**Comment by nickelcheng** [comment]
This PR is OK for me.

File: utils/user_info.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ def has_c_ownership(curr_user, curr_contest):
     except Contest.DoesNotExist:
         logger.warning('Contest id %ld does not exsit!' % curr_contest.id)
 
-    ownership = (curr_user.username == curr_contest.owner)
+    ownership = (curr_user == curr_contest.owner)
```
Comment (bruce3557) [inline utils/user_info.py]: Two points want to check:
(1) In my opinion, ownership -> isOwner is better.
(2) Is ```curr_user``` an user object? (I think previous design is more elegant for the future)

File: utils/user_info.py [hunk 2]
```diff
@@ -54,7 +55,7 @@ def has_g_ownership(curr_user, curr_group):
     except Group.DoesNotExist:
         logger.warning('Group id %ld does not exsit!' % curr_group.id)
 
-    ownership = (curr_user.username == curr_group.owner)
+    ownership = (curr_user == curr_group.owner)
```
Comment (bruce3557) [inline utils/user_info.py]: Ditto

File: utils/user_info.py [hunk 3]
```diff
@@ -70,7 +71,7 @@ def has_p_ownership(curr_user, curr_problem):
     except Problem.DoesNotExist:
         logger.warning('Problem id %ld does not exsit!' % curr_problem.id)
 
-    ownership = (curr_user.username == curr_problem.owner)
+    ownership = (curr_user == curr_problem.owner)
```
Comment (bruce3557) [inline utils/user_info.py]: Ditto

File: utils/user_info.py (hunk 1 above)
Comment (drowsy810301) [inline utils/user_info.py]: Yes,curr_user is an user object.

**Comment by drowsy810301** [comment]
OK

File: utils/user_info.py (hunk 1 above)
Comment (nickelcheng) [inline utils/user_info.py]: curr_user is an user object, curr_contest.owner is an user object, too.
In Bruce's opinion, you can change it to curr_user.username == curr_contest.owner.username

Comment (nickelcheng) [inline utils/user_info.py]: BTW, in python naming rule, it's better to use ```is_owner``` than ```isOwner```

**Comment by drowsy810301** [comment]
Okay,thanks for review.

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**